components/script/dom/readablestreambyobreader.rs

```diff
@@ -81,15 +94,58 @@ impl ReadIntoRequest {
                     );
                 },
             },
+            ReadIntoRequest::ByteTee {
+                byte_tee_read_into_request,
+            } => todo!(),
         }
     }
 
     /// <https://streams.spec.whatwg.org/#ref-for-read-into-request-error-steps>
     pub(crate) fn error_steps(&self, e: SafeHandleValue, can_gc: CanGc) {
-        // error steps, given e
-        // Reject promise with e.
         match self {
-            ReadIntoRequest::Read(promise) => promise.reject_native(&e, can_gc),
+            ReadIntoRequest::Read(promise) => {
+                // error steps, given e
+                // Reject promise with e.
+                promise.reject_native(&e, can_gc)
+            },
+            ReadIntoRequest::ByteTee {
+                byte_tee_read_into_request,
+            } => todo!(),
+        }
+    }
+}
+
+/// The rejection handler for
+/// <https://streams.spec.whatwg.org/#abstract-opdef-readablebytestreamtee>
+#[derive(Clone, JSTraceable, MallocSizeOf)]
+#[cfg_attr(crown, crown::unrooted_must_root_lint::must_root)]
+struct ByteTeeClosedPromiseRejectionHandler {
+    branch_1_controller: Dom<ReadableByteStreamController>,
+    branch_2_controller: Dom<ReadableByteStreamController>,
+    #[ignore_malloc_size_of = "Rc"]
+    canceled_1: Rc<Cell<bool>>,
+    #[ignore_malloc_size_of = "Rc"]
+    canceled_2: Rc<Cell<bool>>,
+    #[ignore_malloc_size_of = "Rc"]
+    cancel_promise: Rc<Promise>,
+}
+
+impl Callback for ByteTeeClosedPromiseRejectionHandler {
+    /// Continuation of <https://streams.spec.whatwg.org/#abstract-opdef-readablebytestreamtee>
+    /// Upon rejection of `reader.closedPromise` with reason `r``,
+    fn callback(&self, _cx: SafeJSContext, v: SafeHandleValue, _realm: InRealm, can_gc: CanGc) {
+        // If thisReader is not reader, return.
+        // TODO: Implement this check.
```

Comment:
For this I think you would have to add the `this_reader: Rc<RefCell<ReaderType>>` and also the `&self` reader from when `forward_reader_error` was called. Could be a TODO indeed, we should note it in the issue.